WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -18,6 +19,7 @@ class AuthenticationManager: Authentication {
                                                                 wpcomSecret: ApiCredentials.dotcomSecret,
                                                                 wpcomScheme: ApiCredentials.dotcomAuthScheme,
                                                                 wpcomTermsOfServiceURL: WooConstants.termsOfServiceUrl.absoluteString,
+                                                                wpcomAPIBaseURL: Settings.wordpressApiBaseURL,
```

Comment:
It is a little odd, ain't it? 

I tried to think of alternatives but came up blank. 🤷‍♂ 

We **are** importing `Networking` in a few places in the WooCommerce package anyway. Not sure why. Also, we are _technically_ using `Networking` models directly in [`Model.swift`](https://github.com/woocommerce/woocommerce-ios/blob/develop/Yosemite/Yosemite/Model/Model.swift). 

That said, this looks like a clean solution to me. 

My suggestion would be to limit the scope and be clear why we're importing `Networking` by importing it like this:

```swift
import struct Networking.Settings
```

Reply:
Ah, good idea to limit the scope. I like that and have updated the branch accordingly.